src/Acme/UserBundle/Entity/User.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Acme\UserBundle\Entity;
+
+use FOS\UserBundle\Model\User as BaseUser;
+use Doctrine\ORM\Mapping as ORM;
+use Doctrine\Common\Collections\ArrayCollection;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(name="fos_user")
+ */
+class User extends BaseUser
+{
+    /**
+     * @ORM\Id
+     * @ORM\Column(type="integer")
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    protected $id;
+
+    /**
+     * @ORM\OneToMany(targetEntity="AppBundle\Entity\Todo", mappedBy="user")
+     */
+    private $todos;
+
+    public function __construct()
+    {
+        $this->todos = new ArrayCollection();
+
+        parent::__construct();
+        // your own logic
+    }
+
+    /**
+     * Add todos
+     *
+     * @param \AppBundle\Entity\Todo $todos
+     * @return User
+     */
+    public function addTodo(\AppBundle\Entity\Todo $todos)
+    {
+        $this->todos[] = $todos;
```

Comment:
Variable name $todos is misleading, use $todo instead

$this->todos is an ArrayCollection, use $this->todos->add($todo) instead